lib/process-services/src/lib/process-comments/mock/comment-process-service.mock.ts

```diff
@@ -18,34 +18,41 @@
 import { Injectable } from '@angular/core';
 import { Observable, from, of } from 'rxjs';
 import { map } from 'rxjs/operators';
-import { CommentModel, UserProcessModel, CommentsService } from '@alfresco/adf-core';
-import { fakeUser1 } from '../mock/comment-process.mock';
+import { CommentModel, CommentsService } from '@alfresco/adf-core';
+import { UserProcessModel } from '@alfresco/adf-process-services';
+
+export const fakeUser1 = { id: 1, email: 'fake-email@dom.com', firstName: 'firstName', lastName: 'lastName' };
```

Comment:
missing `avatarId` which defined as compulsory

Reply:
thanks, good catch, I will fix the missing typings in the future